policies/TrademarkPolicy.md

```diff
@@ -0,0 +1,215 @@
+
+#### Last modified 2022-11-22
+
+
+
+# Purpose
+
+OpenSSL is committed to promoting the use of its open source software. While 
+open source software is generally free to download and modify, the use of
+open source software does not include the right to OpenSSL trademarks.
+
+The Trademark Policy (The Policy) aims to protect and ensure consistent usage 
+of the OpenSSL trademarks and to clarify how the OpenSSL trademarks may be 
+used.
+
+As a part of this process, the OpenSSL trademark is a registered United States 
+trademark of the OpenSSL Software Foundation. 
+For registration outside the United States, we have filed a [Madrid Protocol](http://en.wikipedia.org/wiki/Madrid_system) 
+application to extend the protection in the European Union, China and Japan.
+
+
+The Policy aims to:
+
+1.  to encourage appropriate, authorised use of the OpenSSL trademarks;
+2.  to clarify proper usage of OpenSSL trademarks by third parties;
+3.  to prevent misuse of OpenSSL trademarks that can confuse or mislead
+    users with respect to OpenSSL or its affiliates.
+
+Please note that it is not the goal of The Policy to limit commercial
+activity around OpenSSL. We encourage businesses to work on OpenSSL
+while being compliant with The Policy.
+
+
+# Scope
+
+The Policy encompasses all marks, in word and logo form including but not 
+limited to, OpenSSL name and derivatives thereof, software names, project 
+names, logos and styles collectively referred to as "Our Trademarks". 
+
+The Policy applies to all persons who use or intend to use Our Trademarks.
+
+
+# Principles
+
+* Our Trademarks can only be used in line with The Policy, including but not 
+limited to; use in software, advertising or on websites.
+* Our Trademarks can only be used to identify OpenSSL, its products and 
+services in a way that is accurate, not misleading or disparaging of 
+OpenSSL, its products, services, communities, or users.  
+* Our Trademarks can not be modified, combined with other marks or used as 
+part of a larger mark. 
+* You may not use marks that resemble Our Trademarks, or that may be 
+misleading or cause confusion in the marketplace or implies any affiliation 
+with or endorsement by OpenSSL.
+* Except as provided in The Policy, use of Our Trademarks, is prohibited 
+without express written permission from OpenSSL.  Use without written 
+permission may violate OpenSSL's trademark rights.
+
+
+
+## When Our Trademarks can NEVER be used without permission
+
+Prior written permision must be provided in order to use Our Trademarks in all
+ circumstances except those listed under the "When Our Trademarks can be used without permission" section.
+
+In general:
+1.  You cannot use Our Trademarks in any way that suggests an
+    affiliation with, sponsorship or endorsement by the OpenSSL project or 
+    community, if the same is not true.
+2.  You cannot use Our Trademarks in a company or organization name
+    or as the name of a product or service.
```

Comment:
This is most problematic when talking about a product that is an engine or provider that "plugs in" to OpenSSL where having OpenSSL somewhere in the name makes some sense. We clearly want to avoid the impression that such products are somehow endorsed by OpenSSL, so a name like "XYZ OpenSSL Provider" is difficult because it implies an association between XYZ and OpenSSL. Could we provide a "standard form" name that would be acceptable, e.g "XYZ Provider for OpenSSL"?

Reply:
@mattcaswell "Provider for OpenSSL" also implies an endorsement by us of their product in that they're "our" provider, so not sure what alternative wording we can use but that might be because I'm not sure how the "plugs in" relate to OpenSSL, are these external products that utilise OpenSSL or are they external products that we use within OpenSSL?

Reply:
> "Provider for OpenSSL" also implies an endorsement by us

Does it? It doesn't read that way to me.

> are these external products that utilise OpenSSL or are they external products that we use within OpenSSL

These are third party products - distributed externally - that extend the capabilities of OpenSSL

Reply:
Provider designed for use with OpenSSL ?

Reply:
Would changing point 6 to read: However OpenSSL may grant permission to use Our Trademarks to indicate that an OpenSSL product has been incorporated into another organisation's  product or service, or where another organisation's product or service is designed to work with an OpenSSL product. In these instances the use of conventions such as "compatible with OpenSSL", "based on OpenSSL" or "used with OpenSSL" will apply.

Reply:
It doesn't really cover it because point 2 is explicit that you cannot use OpenSSL in a product name. My idea is that we supply some kind of "template" product name for providers/engines that is always acceptable and you don't need special permission for.

How about

"XYZ Provider compatible with OpenSSL"

A bit wordy perhaps for a name...but might be ok.

Reply:
I thought the point was that we wanted to know who is using our Trademarks and that they have our permission prior to using them, so that we can maintain our ownership of the Trademarks?

With that end, we could amend point 2 to:
You cannot use Our Trademarks in a company or organization name or as the name of a product or service, unless OpenSSL specifically grants you permission to do so.
& then amend point 6 to cover the wording we want for the third party providers.

Would that cover it?

Reply:
> I thought the point was that we wanted to know who is using our Trademarks

No? I don't think this is a requirement.

> that they have our permission prior to using them,

We have a policy which explicitly allows people to use our trademarks without having to ask permission. The point of the policy is to define what those situations are.

If anything I think we want to make it as easy as possible to use our trademarks. The main point is to protect them from inappropriate use - not to place barriers in the way of appropriate use.

Reply:
I think we need to discuss at OMC, as my understanding was that we don't want to restrict appropriate usage but that we did want them to get our permission, in order to be "actively" managing our trademarks, so we don't lose them, but I could be wrong.  I'll add to next OMC agenda.

Reply:
I'm having the same discomfort about this as Matt.  At the same time, I've heard the reasoning for having this and do understand.  That led to my vote today.